tpp/5.0/Dockerfile

```diff
@@ -1,34 +1,39 @@
 ################## BASE IMAGE ######################
-FROM biocontainers/biocontainers:v1.0.0_cv4
+FROM biocontainers/biocontainers
 
 ################## METADATA ######################
-LABEL base_image="biocontainers:v1.0.0_cv4"
-LABEL version="4"
+LABEL base_image="biocontainers"
+LABEL version="5.2"
```

Comment:
This is a new release, it should be in its own directory tpp/5.2/Dockerfile

In a version dir, only bug related fixes on some version are allowed